src/trellis/bundler.py

```diff
@@ -47,6 +47,159 @@ def _safe_extract(tar: tarfile.TarFile, dest: Path) -> None:
+    "@react-aria/progress": "3.4.17",
+    "@react-aria/radio": "3.10.8",
+    "@react-aria/searchfield": "3.7.9",
+    "@react-aria/select": "3.14.10",
+    "@react-aria/selection": "3.20.0",
+    "@react-aria/separator": "3.4.3",
+    "@react-aria/slider": "3.7.12",
+    "@react-aria/ssr": "3.9.6",
+    "@react-aria/switch": "3.6.8",
+    "@react-aria/table": "3.15.4",
+    "@react-aria/tabs": "3.9.6",
+    "@react-aria/tag": "3.4.6",
+    "@react-aria/textfield": "3.14.9",
+    "@react-aria/tooltip": "3.7.8",
+    "@react-aria/utils": "3.25.3",
+    "@react-aria/visually-hidden": "3.8.16",
+    "@react-aria/toolbar": "3.0.0-beta.9",
+    "@react-aria/toggle": "3.10.8",
+    "@react-aria/spinbutton": "3.6.9",
+    "@react-aria/form": "3.0.9",
+    "@react-aria/grid": "3.10.4",
+    "@react-aria/live-announcer": "3.4.0",
+    # React Stately internal packages (required by umbrella)
+    "@react-stately/calendar": "3.5.5",
+    "@react-stately/checkbox": "3.6.9",
+    "@react-stately/collections": "3.11.0",
+    "@react-stately/color": "3.8.0",
+    "@react-stately/combobox": "3.10.0",
+    "@react-stately/data": "3.11.7",
+    "@react-stately/datepicker": "3.10.3",
+    "@react-stately/dnd": "3.4.3",
+    "@react-stately/form": "3.0.6",
+    "@react-stately/list": "3.11.0",
+    "@react-stately/menu": "3.8.3",
+    "@react-stately/numberfield": "3.9.7",
+    "@react-stately/overlays": "3.6.11",
+    "@react-stately/radio": "3.10.8",
+    "@react-stately/searchfield": "3.5.7",
+    "@react-stately/select": "3.6.8",
+    "@react-stately/selection": "3.17.0",
+    "@react-stately/slider": "3.5.8",
+    "@react-stately/table": "3.12.3",
+    "@react-stately/tabs": "3.6.10",
+    "@react-stately/toggle": "3.7.8",
+    "@react-stately/tooltip": "3.4.13",
+    "@react-stately/tree": "3.8.5",
+    "@react-stately/utils": "3.10.4",
+    "@react-stately/flags": "3.0.4",
+    "@react-stately/virtualizer": "4.1.0",
+    "@react-stately/grid": "3.9.3",
+    # Shared types and utilities
+    "@react-types/shared": "3.25.0",
+    "@react-types/button": "3.10.0",
+    "@react-types/checkbox": "3.8.4",
+    "@react-types/calendar": "3.4.10",
+    "@react-types/color": "3.0.0",
+    "@react-types/combobox": "3.13.0",
+    "@react-types/datepicker": "3.8.3",
+    "@react-types/dialog": "3.5.13",
+    "@react-types/grid": "3.2.9",
+    "@react-types/label": "3.9.6",
+    "@react-types/link": "3.5.8",
+    "@react-types/listbox": "3.5.2",
+    "@react-types/menu": "3.9.12",
+    "@react-types/meter": "3.4.4",
+    "@react-types/numberfield": "3.8.6",
+    "@react-types/overlays": "3.8.10",
+    "@react-types/progress": "3.5.7",
+    "@react-types/radio": "3.8.4",
+    "@react-types/searchfield": "3.5.9",
+    "@react-types/select": "3.9.7",
+    "@react-types/slider": "3.7.6",
+    "@react-types/switch": "3.5.6",
+    "@react-types/table": "3.10.2",
+    "@react-types/tabs": "3.3.10",
+    "@react-types/textfield": "3.9.7",
+    "@react-types/tooltip": "3.4.12",
```

Comment:
I assume these aren't included in the bundle?

Reply:
Correct - the `@react-types/*` packages are TypeScript type definitions only. They contain no runtime code and are used by React Aria at compile-time for type checking. esbuild tree-shakes them out of the final bundle since they export only types.